src/components/PortfolioCard/PortfolioCard.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { FC } from '../../typings/components';
+
+import CardMobile from './CardMobile';
+import CardTablet from './CardTablet';
+import CardDesktop from './CardDesktop';
+
+import { CardProps } from './cardProps';
+
+type CardType = 'mobile' | 'tablet' | 'desktop';
+
+interface PortfolioCardProps extends CardProps {
+  type: CardType;
+}
+
+export const PortfolioCard: FC<PortfolioCardProps> = ({ type, description, label, image, title, onClick }) => {
+  const cardProps: CardProps = { description, label, image, title, onClick };
+
+  if (type === 'desktop') return <CardDesktop {...cardProps} />;
+  else if (type === 'tablet') return <CardTablet {...cardProps} />;
```

Comment:
You can use `if` three times (without `else`), as you have return statement in every case

Reply:
I think that the best I can do about it, is to use `if` twice and if it doesn't match, then return for example `mobile` type as a fallback, otherwise, typescript will complain about return type being possible undefined.

Changed to:
```typescript
  if (type === 'desktop') return <CardDesktop {...props} />;
  if (type === 'tablet') return <CardTablet {...props} />;
  return <CardMobile {...props} />;
```

Reply:
Maybe if it was `ENUM`, then typescript wouldn't complain but then, we would have to import the ENUM every time we use it with the component. So I think that the above solution is not bad